dev/tests/static/testsuite/Magento/Test/Php/_files/whitelist/strict_type.txt

```diff
@@ -0,0 +1 @@
+# Format: <componentType=module|library|theme|language|*> <componentName> <globPattern> or simply <globPattern>
```

Comment:
Is it required to have `blacklist` and `whitelist` at the same time?

Reply:
@ishakhsuvarov blacklist removed

Reply:
@ishakhsuvarov, blacklist returned. Yes it is required.

Reply:
There is no value in delivering test which has empty whitelist. I assume it would not test anything.

Reply:
Blacklist deleted.